ci/import-squidex-data.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env sh
+
+# Set to true if you want to print the sq sync logs on success for debugging
+PRINT_SYNC_LOGS_ON_SUCCESS=false
```

Comment:
actually - this should be false by default but we should be able to override it with simple config change (in gitlab) by defining the env variable without having to alter the script

Reply:
Added test for the envvar